exchanges/request-policy/src/requestPolicyExchange.ts

```diff
@@ -9,10 +9,32 @@ import { pipe, tap, map } from 'wonka';
 const defaultTTL = 5 * 60 * 1000;
 
 export interface Options {
+  /** A function allowing you to return a boolean on whether or not to upgrade the
+   * operation to "cache-and-network". */
   shouldUpgrade?: (op: Operation) => boolean;
+  /** The TTL in ms for which we should not upgrade an operation for. */
   ttl?: number;
 }
 
+/** Creates an `Exchange` that can upgrade the request-policy to "cache-and-network" of an urql query operation.
+ *
+ * @param options - An {@link Options} configuration object.
+ * @returns the created request-policy {@link Exchange}.
+ *
+ * @remarks
+ * The `requestPolicyExchange` observes all operations going through and allows you
+ * to upgrade based on a "ttl" or "shouldUpgrade" function.
+ *
+ * @example
+ * ```ts
+ * requestPolicyExchange({
+ *  // Upgrade when we haven't seen this operation for 1 second
+ *  ttl: 1000,
+ *  // or when it's a todos query, we always upgrade those
```

Comment:
```suggestion
 *  // and only upgrade operations that query the `todos` field.
```

Just noting that the exchange does an `&&` not an `||`